demo/inferenceUI/README.md

```diff
@@ -0,0 +1,50 @@
+## KAITO InferenceUI Demo
+
+The KAITO InferenceUI Demo provides a sample front-end application that demonstrates
+how to interface with the KAITO Workspace for inference tasks. 
+This guide covers deploying the front-end as a Helm chart in a Kubernetes environment 
+as well as how to run the Python application independently.
+
+### Prerequisites
+
+- Kubernetes cluster with Helm installed (if using helm chart)
+- Python environment (if running the Python file directly)
+- Access to the KAITO Workspace Service endpoint
```

Comment:
"the IP address of your Kubernetes cluster."  what does it mean?

Reply:
added section